en/docs/tutorials/create-your-first-rest-api.md

```diff
@@ -97,97 +99,182 @@ In this tutorial, let's design the REST API by updating the low-code diagram as
-
-        | **Field**                  | **Value**            |
-        |----------------------------|----------------------|
-        | **Country Code**           | `country`            |
-        | **Response Variable Name** | `populationByCountry`|
-
-    5. Click **Save**.
-
-7. Now let’s extract the population value from the response, calculate the population in millions, and store it in a variable. To do this, follow this procedure:
-
-    1. Click the last **+** icon in the low-code diagram.
-
-    2. Click **Variable** and enter details as follows:
-
-        | **Field**         | **Value**                                         |
-        |-------------------|---------------------------------------------------|
-        | **Variable Type** | `int`                                             |
-        | **Variable Name** | `populationMillions`                              |
-        | Expression        | `(populationByCountry[0]?.value ?: 0) / 1000000`  |
-
-    3. Click **Save**.
-
-8. Now let’s calculate the total COVID-19 case count per million in the population based on the COVID-19 statistics and the population data you have retrieved. Follow this procedure:
-
-    1. Click the last **+** icon in the low-code diagram.
-
-    2. Click **Variable** and enter details as follows:
-
-        | **Field**         | **Value**                                    |
-        |-------------------|----------------------------------------------|
-        | **Variable Type** | `decimal`                                    |
-        | **Variable Name** | `totalCasesPerMillion`                       |
-        | Expression*       | `<decimal>(totalCases / populationMillions)` |
-
-    3. Click **Save**.
-
-9. To build the JSON payload with data of the total cases per million in the population, add a variable.
-
-    Click the last **+** icon in the low-code diagram and click **Variable**. Then enter information as follows:
-
-    | **Field**         | **Value**                     |
-    |-------------------|-------------------------------|
-    | **Variable Type** | `json`                        |
-    | **Variable Name** | `payload`                     |
-    | Expression        | `{country : country, totalCasesPerMillion : totalCasesPerMillion}` |
-
-    Click **Save**.
-
-10. To respond with the JSON payload, add a `Return` statement as follows:
-
-    1. Click the last **+** icon in the low-code diagram and click **Return**.
-
-    2. In the **Return Expression** field, enter `payload`.
-
-    3. Click **Save**.
+   1. Click **+** under **Start**.
+   
+   2. In the **Add Constructs** pane that appears, click **Connector**.
+
+   3. Search for **COVID-19**, and select the **COVID-19 by ballerinax** connector when it appears in the search results. 
+   
+       In the **Endpoint** pane that appears, click **Save** to add the endpoint.
```

Comment:
Seems like another step.

Reply:
The two actions are included in a single step because they both are related to adding this connector. Having a step per action results in too many steps.